test/document_storage_tests/documents_tests/insert_documents_test.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2023 Codenotary Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"testing"
+
+	"github.com/codenotary/immudb/test/document_storage_tests/documents_tests/actions"
+	"github.com/gavv/httpexpect/v2"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/suite"
+)
+
+type InsertDocumentsTestSuite struct {
+	suite.Suite
+	expect     *httpexpect.Expect
+	sessionID  string
+	collection *httpexpect.Object
+}
+
+func (s *InsertDocumentsTestSuite) SetupTest() {
+	s.expect, s.sessionID = actions.OpenSession(s.T())
+	s.collection = actions.CreateCollectionWithNameAndMultipleFields(s.expect, s.sessionID, uuid.New().String())
+}
+
+func (s *InsertDocumentsTestSuite) TestInsertOneDocumentWithMultipleFields() {
+	document := map[string]interface{}{
```

Comment:
this is not a real document, as eventually what you have is map[string]string (all the values will be considered as string). Make it a proper test structure and then json.Marshal